### ISSUE TITLE
Train/val/test split on 300W dataset?

### ISSUE THREAD

**Issue by YuqiHUO** [wywu/LAB#10]
Thank you for your excellent work. Could you provide the split of train/val/test set on 300W dataset? In the paper you said all the data are used in train and other are used in test, you didn't mention validation set.

**Comment by wywu** [comment]
Hi,

We split the 300W dataset as most of the previous literature used to do.
1. training set (3148 images) consisting of AFW (337), LFPW training set (811) and HELEN training set (2000)
2. validation set: (a) the common subset (554) of HELEN testing set (330) and LFPW testing set (224) (b) the challenging subset (135) named IBUG (c) the full set (689 images) consisting of both the common (554) and challenging subsets (135).
3. testing set: the private test-set used for the 300W competition which contains 600 images (300 indoor and 300 outdoor)

Best,
Wayne

**Comment by YuqiHUO** [comment]
Thank you for your reply, so the common subset and challenging subset are both validation set, I know.
